salt/_modules/metalk8s_kubernetes.py

```diff
@@ -132,6 +133,13 @@ def method(manifest=None, name=None, kind=None, apiVersion=None,
                 )
             )
 
+        # Adding label containing metalk8s version (retrieved from saltenv)
+        match = re.search(r'^metalk8s-(?P<version>.+)$', saltenv)
+        if action in ['create', 'replace'] and match:
```

Comment:
Would it make sense to set the label value to `unknown` or something along those lines if there's *no* 'match'?

Reply:
Yes True, I will